### DISCUSSION TITLE
Styling recommendations

### DISCUSSION THREAD

**Discussion by onny** [powerpaul17/nc_money#17] [Ideas]
Hey,
nice to see a banking / finance app for Nextcloud. I also thought about hacking something like this. There's an other cloud software called Cozy and they already have a finance app called "Banks" https://cozy.io/en/features/#bank

Maybe this is a interesting design / concept idea for your app.

Thank you and good luck
Jonas

**Comment by powerpaul17** [comment]
Thanks for the input, I'll have a look at it.. from first glance it looks a lot like what I had in mind.. :smile:
